### ISSUE TITLE
ch6代码注释中对于gaussNewton中雅可比的定义不一致

### ISSUE THREAD

**Issue by YunjuanSUN** [gaoxiang12/slambook2#264]
在代码注释中，对于J的定义是一个vector3d，所以Hession应该等于J*J^T，但是在注释32行注释中写反了。此外在正文128页中对f(x+▲x）展开中，说J^T为f(x)关于x的导数，为n×1的列向量，但应该是J为列向量，转置后为1×n的row向量。

**Comment by JuexiaOfficial** [comment]
确实。如果说以128页的论述为标准，J在这里被认为是行向量，那么注释中就应该是 J*W^{-1}*J^T。普通教材上写二次型一般是x^T*A*x，这里的x是列向量。
